Framework/Algorithms/inc/MantidAlgorithms/FilterEvents.h

```diff
@@ -74,6 +74,8 @@ class DLLExport FilterEvents : public API::Algorithm {
   }
 
 private:
+  friend struct Mantid::Kernel::CreateUsingNew<FilterEvents>;
```

Comment:
Is this used for anything? `CreateUsingNew` is generally used by the `SingletonHolder` classes. I can't see why it is needed here.